### DISCUSSION TITLE
Dialog throws error

### DISCUSSION THREAD

**Discussion by MetalCar** [UI5/webcomponents-react#4604] [Q&A] [answered]
Hey Folks,

is there something i had to take care of when using `<Dialog>`-component?
I often get the following error:
![grafik](https://github.com/SAP/ui5-webcomponents-react/assets/1355298/326e6ee3-8dd6-4181-b550-5947c0150b03)

Implementation is like in the ui5-webcomponents-react StoryBook, with the difference that in my case my dialog content is filled after sending an ajax request.

Can anyone help? (yes i know, i provided no code)

**Comment by Lukas742** [comment]
Hi @MetalCar 

it's not the same error, but could have the same root cause as described here: https://github.com/SAP/ui5-webcomponents/issues/6924

Please also note that we can't really help you if you don't provide a reproducible example or at least describe how we can reproduce the error on our end.

**Reply by MetalCar** [reply]
The problem is, that i'm not able to provide a reproducible example. I tried reproducing it in a codesandbox, but it didn't occures in my examples. I also tried to remove in my code everything that we build on top. But i will have a look into your link. Thank you!

**Reply by MetalCar** [reply]
I have checked the linked issue, but unfortunately it does not help me. It seems that my problem is that when I reload content of the dialog via Ajax, the dialog box then grows to fit the content, but this process does not find an end and the box then continues to grow infinitely until this error occurs. As I said, I can unfortunately not show an example, because I can not reliably recreate this, but I hope I was able to describe the problem well.

**Reply by Lukas742** [reply]
I see that the ui5-webcomponents team provided a fix for the linked issue. 
I'd suggest waiting until the fix is released and if that doesn't solve your issue as well, open an issue in the ui5-webcomponents repo, as the Dialog is developed there and the error you're facing shouldn't be connected to our wrapper.

**Reply by MetalCar** [reply]
Where did you found the fix? I searched in weekend for myself but doesn't found anything. Can you link it here, please?

**Reply by Lukas742** [reply]
https://github.com/SAP/ui5-webcomponents/pull/6934

**Reply by Lukas742** [reply, marked as answer]
I just realized that the fix has already been released with version [1.13.0](https://github.com/SAP/ui5-webcomponents/releases/tag/v1.13.0) of `@ui5/webcomponents`. Sorry about that.
So if you're facing this error with that version as well, please open an [issue](https://github.com/SAP/ui5-webcomponents/issues/new/choose) in the ui5-webcomponents repo.
